java/test/src/main/java/io/ray/test/CrossLanguageInvocationTest.java

```diff
@@ -188,6 +189,85 @@ public void testPassActorHandleFromJavaToPython() {
     Assert.assertEquals(res.get(), "3".getBytes());
   }
 
+  @Test
+  public void testExceptionSerialization() {
+    try {
+      throw new NativeRayException("Test Exception");
+    } catch (NativeRayException e) {
+      try {
+        String formattedException = org.apache.commons.lang3.exception.ExceptionUtils
+            .getStackTrace(e);
+        RayException exception = RayException.parseFrom(e.toBytes());
+        Assert.assertEquals(exception.getFormattedExceptionString(), formattedException);
+      } catch (InvalidProtocolBufferException ex) {
```

Comment:
You can just add `throws InvalidProtocolBufferException` to the test method and remove the try-catch.

Reply:
Got it. Thanks.